packages/web/src/components/GraphQLHooksProvider.tsx

```diff
@@ -24,14 +32,25 @@ type DefaultUseSubscriptionType = <
   subscription: DocumentNode,
   options?: GraphQLSubscriptionHookOptions<TData, TVariables>
 ) => SubscriptionOperationResult<TData, TVariables>
+
+type DefaultUseSuspenseType = <
+  TData = any,
+  TVariables extends OperationVariables = GraphQLOperationVariables
+>(
+  query: DocumentNode,
+  options?: GraphQLQueryHookOptions<TData, TVariables>
+) => SuspenseQueryOperationResult<TData, TVariables>
+
 export interface GraphQLHooks<
   TuseQuery = DefaultUseQueryType,
   TuseMutation = DefaultUseMutationType,
-  TuseSubscription = DefaultUseSubscriptionType
+  TuseSubscription = DefaultUseSubscriptionType,
```

Comment:
I wouldn't surprise me that we will need to add more hooks like useBackgroundQuery these: https://www.apollographql.com/docs/react/api/react/hooks/#usebackgroundquery

So, this sets that up by having more that just query, subscription and mutations.

Reply:
Yeah possibly..... Remember I think the HooksProvider was originally setup so that you could swap Apollo for something else. `useBackgroundQuery` and `useReadQuery`, etc. may not exist in other clients...